post_processing/plot/plot_helpers.jl

```diff
@@ -415,6 +418,17 @@ function generate_paperplots_held_suarez(fig_dir, nc_files; moist)
             if moist
                 global qt = nc["qt"][:]
             end
+            if haskey(nc, "sfc_flux_energy")
+                sfc_flux_energy = nc["sfc_flux_energy"][:]
+            end
+            if haskey(nc, "sfc_evaporation")
+                sfc_evaporation = nc["sfc_evaporation"][:]
+            end
+            if haskey(nc, "lw_flux_up")
+                # NOTE: diff = up - down
+                lw_flux_diff = nc["lw_flux_up"][:] - nc["lw_flux_down"][:]
```

Comment:
```suggestion
                lw_flux_net = nc["lw_flux_up"][:] - nc["lw_flux_down"][:]
```
(just a suggestion, as `net flux` is commonly used in atmosphere science.)

Reply:
I'll change this, but to explain my reasoning: I used `diff` because there's a sign difference between surface/top of atmosphere (which is resolved in the plotting stage below).

Reply:
Yes, this makes more sense when I saw the comment in the plotting. I don't have a strong preference here, so feel free to change it back if that makes more sense to you.